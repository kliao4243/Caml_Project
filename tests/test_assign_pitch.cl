int main(){
    pitch a = 2b4;
    pitch b = 3^7;
    pitch c = 4#5;
    print(a);
    print(b);
    print(c);
}
