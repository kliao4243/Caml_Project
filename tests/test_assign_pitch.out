2b4
3^7
4#5
